onnx_tf/handlers/backend/mat_mul_integer.py

```diff
@@ -0,0 +1,35 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_func
+
+
+@onnx_op("MatMulInteger")
+@tf_func(tf.matmul)
+class MatMulInteger(BackendHandler):
+
+  @classmethod
+  def version_10(cls, node, **kwargs):
+    tensor_dict = kwargs["tensor_dict"]
+    A = tensor_dict[node.inputs[0]]
+    B = tensor_dict[node.inputs[1]]
+    a_zero_point = tensor_dict[
```

Comment:
Can we change the code around so the graph is much smaller when no zero_points are given?

Reply:
done, @chinchuang007 may you please review again, thanks